src/util.js

```diff
@@ -153,8 +153,12 @@ var _ = $.extend(Mavo, {
 	/**
 	 * Get the current value of a CSS property on an element
 	 */
-	getStyle: (element, property) => element && getComputedStyle(element).getPropertyValue(property).trim(),
-
+    getStyle: function(element, property) {
+        var value = getComputedStyle(element).getPropertyValue(property);
+        if (value) {
+            return element && value.trim();
+        }
+    },
```

Comment:
Please insert a blank line before the if (blank lines before control flow statements is part of the coding style).
Btw arrow functions are fine, they are transpiled automatically in the *.es5.js files.